specification/securityinsights/resource-manager/Microsoft.SecurityInsights/preview/2021-03-01-preview/operations.json

```diff
@@ -0,0 +1,117 @@
+{
+    "swagger": "2.0",
+    "info": {
+      "title": "Security Insights",
+      "description": "API spec for Microsoft.SecurityInsights (Azure Security Insights) resource provider",
+      "version": "2021-03-01-preview"
+    },
+    "host": "management.azure.com",
+    "schemes": [
+      "https"
+    ],
+    "consumes": [
+      "application/json"
+    ],
+    "produces": [
+      "application/json"
+    ],
+    "security": [
+      {
+        "azure_auth": [
+          "user_impersonation"
+        ]
+      }
+    ],
+    "securityDefinitions": {
+      "azure_auth": {
+        "type": "oauth2",
+        "authorizationUrl": "https://login.microsoftonline.com/common/oauth2/authorize",
+        "flow": "implicit",
+        "description": "Azure Active Directory OAuth2 Flow",
+        "scopes": {
+          "user_impersonation": "impersonate your user account"
+        }
+      }
+    },
+    "paths": {
+        "/providers/Microsoft.SecurityInsights/operations": {
+            "get": {
+              "operationId": "Operations_List",
+              "description": "Lists all operations available Azure Security Insights Resource Provider.",
+              "parameters": [
+                {
+                  "$ref": "../../../common/v1/types.json#/parameters/ApiVersion"
+                }
+              ],
+              "produces": [
+                "application/json"
+              ],
+              "responses": {
+                "200": {
+                  "description": "OK. Successfully retrieved operations list.",
+                  "schema": {
+                    "$ref": "#/definitions/OperationsList"
+                  }
+                }
+              },
+              "x-ms-pageable": {
+                "nextLinkName": "nextLink"
+              }
+            }
+          }
+    },
+    "definitions": {
+        "OperationsList": {
+            "description": "Lists the operations available in the SecurityInsights RP.",
+            "properties": {
+              "nextLink": {
```

Comment:
>"nextLink" [](start = 14, length = 10)

Mark this property as readOnly: true, as a best practice, annotate all the properties that your service sets and are not meant to be updated by end users, with readOnly : true.